src/main/java/uk/gov/companieshouse/company/links/processor/InsolvencyStreamProcessor.java

```diff
@@ -65,9 +70,19 @@ public void processDelete(Message<ResourceChangedData> resourceChangedMessage) {
             return;
         }
 
-        links.setInsolvency(null);
+        final ApiResponse<CompanyInsolvency> companyInsolvencyResponse = companyInsolvencyService
+                .getCompanyInsolvency(logContext, companyNumber);
         CompanyProfile companyProfile = new CompanyProfile();
```

Comment:
This line can be moved into if block

Reply:
Can we discuss this please ? as have some query regarding this comment

Reply:
Thanks